frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerThread.java

```diff
@@ -80,6 +80,7 @@ public class WorkerThread implements Runnable {
     private WorkerLifeCycle lifeCycle;
     private int responseTimeout;
     private long recoveryStartTS; // 0: default value. no recovery needed, in healthy mode
+    private String telemetryPath = System.getenv("SM_TELEMETRY_LOG_REV_2022_12");
```

Comment:
Do we have this env variable documented somewhere? Or is this SM internal only? We are not parsing the value here and just assuming non-null value `== true`. What if a customer sets this to `False` explicitly?

Reply:
This is a SM internal only. We use `SM_TELEMETRY_LOG_REV_2022_12 ` to avoid collision with customer defined env variable. In the case of collision or customer set this environment variable by accident, Remote appender will fail (https://github.com/aws/sagemaker-pytorch-inference-toolkit/pull/146) and we are not going to see Telemetry log on our end.